### DISCUSSION TITLE
Locked Properties

### DISCUSSION THREAD

**Discussion by Serviettenknoedel** [marcusolsson/obsidian-projects#675] [Help]
Hi!

I would like to visualize my files through Projects and filter by tags. Is there a funktion where I can lock file-editing in a view?

My issue is that it is very easy to manipulate files/metadata in e.g. tableview (what it is meant to be), but I would like to have a "reading view" where it is not possible to change properties unintended.

Is there a funktion for that or is something like this planned?

Thanks,
Serviettenknoedel

**Comment by Serviettenknoedel** [comment]
Hello again.

Is there anyone who can help me?

Greetings,
Serviettenknoedel

**Comment by Acylation** [comment]
Hi! Thanks for following up. I was stuck by the new feature, and sorry for my delay on reply.

The feature would be useful, and perhaps straightforward to implement, given that we already have readonly views built for the Dataview-sourced projects. Are you expecting a reading mode for each individual view or for the entire project?

Please note that, when we toggle to the potential reading mode for board or calendar views, the drag&drop feature will be disabled, as moving items to different places will also change the file property. Would this design still meet your need?

**Comment by Serviettenknoedel** [comment]
Hi Acylation!

Thanks for your answer.

How can I toggle the read only view?

To answer your question: I would love to use Obsidian Projects for my knowledge archive. For that, it is important to me that it is not possible to change/delete properties unintendet. Ideally for each view and optionally the entire project. 

A solution for making it work would be for example with a simple button "read only" or "safe mode". An option in the Projects-settings like a toggle for "safe mode - confirm each change of properties" would be awesome.

Greetings, Serviettenknoedel
